client/wfnews-war/src/main/angular/src/app/utils/index.ts

```diff
@@ -368,3 +368,25 @@ export function convertToDateTimeTimeZone(date) {
     }
     return convertedDate;
 }
+
+export function setDisplayColor(stageOfControlCode: string) {
```

Comment:
Good idea. I think this gets duplicated in a few spots on the UI. We could probably add it to an scss import too